test/e2e/node/kubelet.go

```diff
@@ -310,52 +302,73 @@ var _ = SIGDescribe("kubelet", func() {
 			}
 		})
 
-		for _, itArg := range deleteTests {
-			name := fmt.Sprintf(
-				"kubelet should be able to delete %d pods per node in %v.", itArg.podsPerNode, itArg.timeout)
-			ginkgo.It(name, func() {
-				totalPods := itArg.podsPerNode * numNodes
-				ginkgo.By(fmt.Sprintf("Creating a RC of %d pods and wait until all pods of this RC are running", totalPods))
-				rcName := fmt.Sprintf("cleanup%d-%s", totalPods, string(uuid.NewUUID()))
-
-				err := framework.RunRC(testutils.RCConfig{
-					Client:       f.ClientSet,
-					Name:         rcName,
-					Namespace:    f.Namespace.Name,
-					Image:        imageutils.GetPauseImageName(),
-					Replicas:     totalPods,
-					NodeSelector: nodeLabels,
-				})
-				framework.ExpectNoError(err)
-				// Perform a sanity check so that we know all desired pods are
-				// running on the nodes according to kubelet. The timeout is set to
-				// only 30 seconds here because framework.RunRC already waited for all pods to
-				// transition to the running status.
-				err = waitTillNPodsRunningOnNodes(f.ClientSet, nodeNames, rcName, ns, totalPods, time.Second*30)
-				framework.ExpectNoError(err)
-				if resourceMonitor != nil {
-					resourceMonitor.LogLatest()
-				}
+		ginkgo.It("kubelet should be able to delete %d pods per node in %v", func() {
+			totalPods := 10 * numNodes
+			ginkgo.By(fmt.Sprintf("Creating a RC of %d pods and wait until all pods of this RC are running", totalPods))
+			rcName := fmt.Sprintf("cleanup%d-%s", totalPods, string(uuid.NewUUID()))
+
+			err := framework.RunRC(testutils.RCConfig{
+				Client:       f.ClientSet,
+				Name:         rcName,
+				Namespace:    f.Namespace.Name,
+				Image:        imageutils.GetPauseImageName(),
+				Replicas:     totalPods,
+				NodeSelector: nodeLabels,
+			})
+			framework.ExpectNoError(err)
+			// Perform a sanity check so that we know all desired pods are
+			// running on the nodes according to kubelet. The timeout is set to
+			// only 30 seconds here because framework.RunRC already waited for all pods to
+			// transition to the running status.
+			err = waitTillNPodsRunningOnNodes(f.ClientSet, nodeNames, rcName, ns, totalPods, time.Second*30)
```

Comment:
Nit: I'd define `testTimeout := time.Second*30` up at the top of the function block with `totalPods`, so it is more readable how long the test has to execute.